### DISCUSSION TITLE
How do I determine the compatibility of nerdctl and containerd versions

### DISCUSSION THREAD

**Discussion by pptfz** [containerd/nerdctl#2604] [General]
I checked the official documentation, but I couldn't find any compatibility notes for the nerdctl version and the containerd version
When I install an arbitrary version of containerd, how do I know which nerdctl versions I should install

**Comment by fahedouch** [comment]
we recommend using the latest version of nerdctl. Compatible containerd versions are mentioned in the release notes ( e.g [1.6.2 ](https://github.com/containerd/nerdctl/releases/tag/v1.6.2))

**Reply by pptfz** [reply]
Thank you very much
